### PR TITLE
{bio}[foss/2018b] CompareM v0.0.23 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/2954b25db911cf66c097ad228a2bbadf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3158.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/a936e49027bb792a3fc11cd6aca4202e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2419.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5a6a41686697a9f4bf23de2ad63a2345 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/78241926fd77079c49745e4ae671d128 for a full test report.

File: easybuild/easyconfigs/c/CompareM/CompareM-0.0.23-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+exts_list = [
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('biolib', '0.0.46', {
+        'checksums': ['232e0d02d4b0a759f94c613f22ab3a9e5bb9230666308446fa8e08c7597db156'],
+    }),
+    ('comparem', version, {
+        'checksums': ['1f4b693298d92499bf7c3174220e06260e3c446273517c43781b14b284155f5b'],
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/CompareM/CompareM-0.0.23-foss-2018b-Python-2.7.15.eb]: The install instructions for comparem sys that it assumes
prodigal and diamond to be in the path.

They are missing here.

**Review by akesandgren — DISMISSED** [review]

File: easybuild/easyconfigs/c/CompareM/CompareM-0.0.23-foss-2018b-Python-2.7.15.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/c/CompareM/CompareM-0.0.23-foss-2018b-Python-2.7.15.eb:31]: fixed in 644260d

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2602.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/21fbb55d51668b1a703f4d7f7d2a1dc4 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/fbd2eae0aa0f039c397115596de7acec for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**